lib/datadog/appsec/contrib/devise/ext.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AppSec
+    module Contrib
+      module Devise
+        # Devise integration constants
+        module Ext
+          APP = 'devise'
```

Comment:
Is this constant used anywhere?

Reply:
None of the `Ext::APP` constants are used at all. We can decide to remove them all, but since is the standard I prefer to remove it on a separate PR

Reply:
Yeah we should remove them (and any other unused constant), they're confusing as hell.